src/wbi/commands/flash_finder.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import logging
+from wbi.legacy.flash_finder import flash_finder
+
+logger = logging.getLogger(__name__)
+
+
+def add_args(parser):
+    parser.add_argument(
+        "--input_folder",
+        type=str,
+        required=True,
+        help="Path to folder containing data files",
+    )
+    parser.add_argument(
+        "--output_folder",
+        type=str,
+        required=True,
+        help="Path to the folder to save output files",
+    )
+    parser.add_argument(
+        "--chunksize", type=int, required=False, help="Approximate number of frames"
+    )
+    parser.add_argument(
+        "-v", "--verbose", action="store_true", help="Increase verbosity"
```

Comment:
This is not used anywhere. Just set the logging level appropriately here based on this flag, just like the current hello/goodbye commands, before those 2 sample commands are gone.

Reply:
Added a very basic logging level information.